src/components/toast/toast.style.ts

```diff
@@ -24,6 +25,25 @@ const StyledPortal = styled(Portal)<{
       transform: translateX(-50%);
     `}
 
+    ${align === "left" &&
+    css`
+      margin-left: 5%;
+      transform: translateX(-50%);
+    `}
```

Comment:
**issue**: when `align="left"`, it seems the `Toast` goes a little too far off screen on the left hand side.

We may need to adjust the css rules a bit? 

<img width="215" alt="Screenshot 2023-08-18 at 13 22 22" src="https://github.com/Sage/carbon/assets/18368713/f1b52b3b-5887-451f-ad36-b320223fe66a">